lib/spawn/supervisor.ex

```diff
@@ -21,7 +21,7 @@ defmodule Spawn.Supervisor do
   @impl true
   def init(config) do
     children = [
-      Spawn.Cluster.StateHandoff.Supervisor,
+      Spawn.Cluster.StateHandoff.Supervisor.child_spec(config),
```

Comment:
I wonder if this is the correct sequence or if it makes more sense to raise the supervisor cluster before the StateHandoff. As you are now using Node.list it would be right to put the StateHandoff after the cluster_supervisor. What do you think ?

Reply:
I'm not sure if it will break anything though...
I havent tested it but it makes sesne for sure